tests/testProjectedLikelihood.py

```diff
@@ -106,7 +107,8 @@ def setUp(self):
         self.t01 = lsst.meas.modelfit.LocalUnitTransform(self.position, self.sys0, self.sys1)
         self.bbox1 = lsst.afw.geom.Box2I(self.bbox0)
         self.bbox1.grow(-60)
-        self.footprint1 = lsst.afw.detection.Footprint(self.bbox1)
+        self.spanSet1 = lsst.afw.geom.SpanSet(self.bbox1)
+        self.footprint1 = lsst.afw.detection.Footprint(self.spanSet1)
```

Comment:
Is there a good reason that a new ``Footprint`` can't be initialized with a ``Box2I`` and create the ``SpanSet`` itself?  It seems like an unnecessary step to require the user to do it, since (in this case and likely the most common usage) ``spanSet1`` is never used outside of the footprint.